data/test/scenarios/premature_end_turn.cfg

```diff
@@ -0,0 +1,46 @@
+# wmllint: no translatables
+
+# If we were designing the API anew, then [end_turn] before side 1 has even started its turn would
+# probably be treated as an error. However, it's worked this way for ages and has been tested to
+# work this way since 1.13.11.
+
+#####
+# API(s) being tested: [event], [end_turn]
```

Comment:
I've not been adding a space after the comma, so at least for consistency it should be removed here as well. Nothing is actually trying to parse the values listed here though so in practice it doesn't really matter.

Reply:
Ah, I'd missed that.

Without the comma, how do we separate tags which depend on their parent? For example, #7157 has tests for two different `[special_note]` tags:
* `[effect][special_note], [store_unit], [unstore_unit]`
* `[movetype][special_note], [store_unit], [unstore_unit]`

Reply:
Oh, I misread your comment as "I've not been adding the comma". It makes sense now I've realised what it actually says; new version will be pushed in a half-hour or so.